app-prefix/README.md

```diff
@@ -0,0 +1,46 @@
+---
+creation_date: 2024-09-26
+issues:
+- https://github.com/giantswarm/workload-clusters-fleet/pull/802
+last_review_date: 2024-09-26
+owners:
+- https://github.com/orgs/giantswarm/teams/team-honeybadger
+state: review
+summary: Decide whether the apps in CAPI cluster should have or not a prefix, and if we need to enforce it.
+---
+
+# App prefix
+
+## Intro
+
+When we moved to Cluster API, we decided that all resources would live the organization namespace. That decision means apps, including the cluster app, from different clusters coexist in the same namespace. It also means, it forces us to name the apps differently to avoid conflicts.
```

Comment:
@pipo02mix a bit more context is needed here as well:

In the future we would like to separate apps and clusters. Meaning that a development team has access on the management cluster to deploy and manage their app but not a cluster. Maybe even the app can be migrated automatically between clusters by the platform team. So independence between clusters and apps becomes more important.

Reply:
yeah @piontec will help me (next week). I think it can influence the decision, but still we can do what you describing having or not the prefix